extensions/wikia/Our404Handler/SpecialOur404Handler_body.php

```diff
@@ -83,7 +83,9 @@ public function doRender404() {
 		if( !is_null( $oTitle ) ) {
 			// Preserve the query string on a redirect
 			$query = parse_url ( $_SERVER['REQUEST_URI'], PHP_URL_QUERY );
-			if( $namespace == NS_SPECIAL || $namespace == NS_MEDIA ) {
+			if( $namespace == NS_SPECIAL || $namespace == NS_MEDIA
+				|| ( $namespace && $wgOur404HandlerBroaderRedirects )
```

Comment:
$namespace would be null for no ":" in URL and 0 for NS_MAIN for URLs with ":" in it.

In both cases we want this to evaluate to false.

Point taken it doesn't look like I know what I'm doing here.

Will work on making it clearer for the permanent fix.